src/test/java/co/rsk/bitcoinj/script/P2shErpFederationRedeemScriptParserTest.java

```diff
@@ -12,11 +16,25 @@ public class P2shErpFederationRedeemScriptParserTest {
 
     private List<BtcECKey> defaultRedeemScriptKeys;
     private List<BtcECKey> emergencyRedeemScriptKeys;
+    private static final long VALID_CSV_VALUE = 300L;
```

Comment:
Let's use mainnet value to make it realistic

Reply:
Updated.